### PR TITLE
Fix for issue #234 and creature damage taken timer to imo improve evade

### PR DESCRIPTION
**Pull request by vijuxamk**

### CODE REVIEW

File: game/world/managers/objects/units/creature/CreatureManager.py [hunk 1]
```diff
@@ -533,8 +533,16 @@ def _perform_random_movement(self, now):
 
     def _perform_combat_movement(self):
         if self.combat_target:
+
+            # Fixes issue #234
+            # If target is dead > evade
+            if self.combat_target.is_alive == False:
```
Comment (devwar87) [inline game/world/managers/objects/units/creature/CreatureManager.py:539]: There might be more targets attacking this unit, check attack_update() first lines.

File: game/world/managers/objects/units/UnitManager.py [hunk 2]
```diff
@@ -154,6 +154,7 @@ def __init__(self,
         self.object_type_mask |= ObjectTypeFlags.TYPE_UNIT
         self.update_packet_factory.init_values(UnitFields.UNIT_END)
 
+        self.time_since_damage_taken = 0
```
Comment (devwar87) [inline game/world/managers/objects/units/UnitManager.py:157]: From 0.5.4 patch notes:
- Creature pursuit is now timer based rather than distance based.

**Comment by GrenderG** [comment]
Hello! First of all, thanks a lot for your contribution, it's always nice to see new people around here. However, as @devwar87 said, according to [0.5.4 patch notes](https://wowpedia.fandom.com/wiki/Patch_0.5.4), the creature pursuit mechanic worked based on distance rather than on timers in 0.5.3.

```
Combat
* Creature pursuit is now timer based rather than distance based.
```

Blizzard was a bit against kiting mechanics back in the early days.

File: game/world/managers/objects/units/creature/CreatureManager.py (hunk 1 above)
Comment (vijuxamk) [reply game/world/managers/objects/units/creature/CreatureManager.py:539]: Oh yes, here I was too eagerly pursuing my first ever push to open source.. Gonna check that out.

Comment (devwar87) [reply game/world/managers/objects/units/creature/CreatureManager.py:539]: You can still fix #234 👍 , just forget the timer thing.

**Comment by GrenderG** [comment]
You can ping any of us on Discord if you want @vijuxamk, we can talk about fixing #234.

**Pull request closed without merging**